boards/sky-drones/smartap-airlink/init/rc.board_mavlink

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# board specific MAVLink startup script.
+#------------------------------------------------------------------------------
+
+# Start MAVLink on the USB port
+mavlink start -d /dev/ttyACM0
```

Comment:
This can be skipped if you have VBUS. We monitor USB and only start the CDCACM driver once it's connected, and then based on actual serial traffic automatically start mavlink USB when we see a HEARTBEAT. Alternatively you can also launch NSH with 3 newlines, or use it as bridge between the GPS and ucenter.

Reply:
Thanks, makes sense! Yes, we have VBUS. Now removed.